src/components/Avatar/Avatar.js

```diff
@@ -1,13 +1,23 @@
 import React, { memo } from 'react';
 import { string } from 'prop-types';
 import { addShadow } from '../../utils';
+import { runtimeConfig } from '../../config';
 
 import './Avatar.css';
 
 function Avatar(props) {
   const { src, srcSet, alt } = props;
+  const avatarSize = runtimeConfig.AVATAR_SIZE || null;
 
-  return (
+  return avatarSize ? (
+    <img
```

Comment:
What if we did something like


```js
    <img
      className={'avatar' + addShadow()}
      src={src}
      srcSet={srcSet}
      alt={alt}
      style={avatarSize ?{ width: avatarSize, height: avatarSize }: {}}
    />
```
That way we don't have to repeat the component and conditionally render it, we will just give it a style if it does?

Reply:
updated pr. Thanks for review.